SU2_CFD/include/numerics_structure.hpp

```diff
@@ -1537,7 +1543,7 @@ class CUpwCUSP_Flow : public CNumerics {
  */
 class CUpwRoeBase_Flow : public CNumerics {
 protected:
-  bool implicit, grid_movement, roe_low_dissipation;
+  bool implicit, dynamic_grid, roe_low_dissipation;
```

Comment:
From the perspective of making live easier for people who are developing these classes, is the variable name change important?

Reply:
Given that I changed the method to `GetDynamic_Grid()`, I think that renaming the variable makes things more consistent.

Reply:
Fair enough, I only mentioned it because I think there are 4 PR's at the moment with changes to this type of booleans, let's just make sure things are kept consistent after all is merged.

Reply:
That comment was meant to go with the rest of the review :D Yeah, I agree, let's keep it consistent.